admin/securedrop_admin/__init__.py

```diff
@@ -200,6 +200,14 @@ def validate(self, document):
                 message=("Must contain a @ and be set to "
                          "something other than ossec@ossec.test"))
 
+    class ValidateSASLDomain(Validator):
+        def validate(self, document):
+            if document.text == '' or SiteConfig.ValidateFQDN():
```

Comment:
It has been decided to not validate the domain (i.e. not try to resolve them) because we're not able to make that work reliably in Tails at this moment. Reason why there is no validation for the SASLDomain.

Reply:
Should I keep this class as-is nonetheless? I would have to create a new class anyways (e.g. `ValidateNothing`) since every other field is validated and no existing class satisfies the validation requirements. I would also remove the validation alltogether but it feels like a step backwards.

Reply:
You can set it to **None** and it will not be validated at all.